local_units/management/commands/notify_validators.py

```diff
@@ -0,0 +1,60 @@
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.template.loader import render_to_string
+from django.utils import timezone
+from sentry_sdk.crons import monitor
+
+from local_units.models import LocalUnit
+from local_units.utils import (
+    get_email_context,
+    get_global_validators,
+    get_region_admins,
+)
+from main.sentry import SentryMonitor
+from notifications.notification import send_notification
+
+
+class Command(BaseCommand):
+    help = "Notify validators for the pending local units in different period of time"
+
+    @monitor(monitor_slug=SentryMonitor.NOTIFY_VALIDATORS)
+    def handle(self, *args, **options):
+        self.stdout.write(self.style.NOTICE("Notifying the validators..."))
+
+        # Regional Validators: 14 days
+        queryset_for_regional_validtors = LocalUnit.objects.filter(
+            validated=False, is_deprecated=False, created_at__date=timezone.now() - timedelta(days=14)
+        )
+
+        # Global Validators: 28 days
+        queryset_for_global_validators = LocalUnit.objects.filter(
+            validated=False, is_deprecated=False, created_at__date=timezone.now() - timedelta(days=28)
+        )
```

Comment:
We should use `greater than or equal to` comparison instead of `is equal to` for the dates